packages/@okta/vuepress-site/docs/reference/api/push-providers/index.md

```diff
@@ -0,0 +1,625 @@
+---
+title: Push Providers
+category: management
+---
+
+# Push Providers API
+
+<ApiLifecycle access="ie" />
+
+> **Note:** This feature is only available as a part of Okta Identity Engine. [Contact support](mailto:dev-inquiries@okta.com) for further information.
+
+The Okta Push Providers API provides a centralized integration platform to fetch and manage push provider configurations. Okta administrators can use these APIs to provide their push provider credentials, for example from APNs and FCM, so that Okta can send push notifications to their own custom push authenticator applications.
+
+The Push Providers API supports the following **Authorization Schemes**:
+* SSWS - [API tokens](/docs/reference/core-okta-api/#authentication)
+* Bearer - [OAuth2.0 and OpenID Connect](/docs/concepts/oauth-openid/)
+
+## Get started
+
+Explore the Push Providers API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/83575c0b5b075783862c)
+
+## Push Providers operations
+
+The Push Providers API has the following CRUD operations:
+
+* [Create Push Provider](#create-push-provider)
+* [List Push Providers](#list-push-providers)
+* [Get Push Provider by ID](#get-push-provider-by-id)
+* [Update Push Provider](#update-push-provider)
+* [Delete Push Provider](#delete-push-provider)
+
+### Create Push Provider
+
+<ApiOperation method="post" url="/api/v1/push-providers" />
+
+Creates a Push Provider. Each Push Provider must have a unique `name`.
+
+#### Permitted OAuth 2.0 scopes
```

Comment:
I think it's very useful to have the scope or scopes that an operation requires available like this. I know that the Devices API docs uses the term "permitted", but I don't think it's accurate. The MyAccount API docs use the term "Required scope". What do you think?
```suggestion
#### Required OAuth 2.0 scopes
```